src/static/css/footer.less

```diff
@@ -240,7 +240,7 @@
         }
     }
 
-    .list_link__disabled {
```

Comment:
Why is this still here? Why are and of the disabled-link things in this file still here?